lightning/restore/tidb_test.go

```diff
@@ -224,6 +225,25 @@ func (s *tidbSuite) TestInitSchemaSyntaxError(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (s *tidbSuite) TestInitSchemaErrorLost(c *C) {
+	ctx := context.Background()
+
+	s.mockDB.
+		ExpectExec("CREATE DATABASE IF NOT EXISTS `db`").
+		WillReturnResult(sqlmock.NewResult(1, 1))
+	s.mockDB.
+		ExpectClose()
```

Comment:
actually you could do

```go
s.mockDB. 
    ExpectExec("create table.*").
    WillReturnError(errors.New(...))
```

to inject an error. no need to introduce a failpoint
```